gubernator/static/build.js

```diff
@@ -99,13 +100,54 @@ function expand_all(btn) {
 	btn.remove();
 }
 
+/* given a string containing ansi formatting directives, return a new one
+   with designated regions of text marked with the appropriate color directives,
+   and with all unknown directives stripped */
+function ansi_to_html(orig) {
+	function annotate(p1, p2, string) {
+		var p1n = +(p1.replace('0;', ''));
+		if (p1n == 0) // reset
+			return p2;
+		else if (p1n === 1) // bold
+			return '<em>' + p2 + '</em>';
```

Comment:
do other browsers handle weights between 400 and 700 well? You could consider something along those lines.